src/utils/components/NetworkInterfaceModal/utils/helpers.ts

```diff
@@ -40,6 +40,14 @@ export const updateVMNetworkInterface = (
   return updatedVM;
 };
 
+export const markInterfaceAbsent = (interfaces: V1Interface[], nicName: string) => {
+  if (!interfaces.find((iface) => iface?.name === nicName)) return null;
+  const updatedInterfaces = produce<V1Interface[]>(interfaces, (draftInterfaces: V1Interface[]) => {
+    draftInterfaces.find((iface) => iface.name === nicName).state = 'absent';
```

Comment:
please make it more clear

Reply:
I made some changes to try to make it easier to read. Let me know what you think.